packages/connect-explorer/package.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "@trezor/connect-explorer",
+    "version": "1.0.0",
+    "author": "Trezor <info@trezor.io>",
+    "description": "Simple UI for trezor-connect examples",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/trezor/trezor-suite.git"
+    },
+    "bugs": {
+        "url": "https://github.com/trezor/trezor-suite/issues"
+    },
+    "license": "LGPL-3.0+",
+    "scripts": {
+        "dev": "webpack-dev-server --config ./webpack/dev.babel.js --mode development",
+        "build": "rm -rf build && webpack --config ./webpack/prod.babel.js --progress",
+        "lint": "eslint '**/*{.ts,.tsx}'",
+        "type-check": "tsc --noEmit"
+    },
+    "dependencies": {
+        "markdown-it": "^8.4.2",
+        "markdown-it-link-attributes": "^2.1.0",
+        "markdown-it-replace-link": "^1.0.1",
+        "react": "17.0.1",
+        "react-dom": "17.0.1",
+        "react-infinity-menu": "^4.0.0",
+        "react-inspector": "^2.3.1",
+        "react-json-tree": "^0.11.0",
+        "react-redux": "^7.2.2",
+        "react-router": "^5.2.0",
+        "react-router-dom": "^5.2.0",
+        "redux": "^4.0.5",
+        "redux-logger": "^3.0.6",
+        "redux-thunk": "^2.3.0",
+        "stringify-object": "^3.3.0",
+        "styled-components": "^5.3.3",
+        "trezor-connect": "8.2.6-beta.2"
+
+    },
+    "devDependencies": {
+        "@types/markdown-it": "12.2.3", 
+        "@types/markdown-it-link-attributes": "3.0.1",
+        "@types/styled-components": "^5.1.7",
+        "@types/react-router": "^5.1.11",
+        "@types/react-router-dom": "^5.1.7",
+        "babel-loader": "^8.2.1",
+        "html-webpack-plugin": "^5.3.2",
+        "eslint-config-airbnb-base": "^14.2.1",
```

Comment:
are these `eslint` deps really needed?

Reply:
maybe no, I can try to unify eslint config after your PR is merged.

Reply:
removed.